Makefile

```diff
@@ -52,6 +52,7 @@ clean:
 format:
 	isort .
 	black --config pyproject.toml .
+	flake8 .
```

Comment:
This is to fix some things that `make test` can still complain about, even after a `make format`? (I've had that happen to me)